demo/app/app.component.spec.ts

```diff
@@ -1,25 +1,31 @@
-import { async, TestBed } from '@angular/core/testing';
+import { async, TestBed, ComponentFixture } from '@angular/core/testing';
 import { RouterTestingModule } from '@angular/router/testing';
 
 import { AppComponent } from './app.component';
+import { Component } from '@angular/core';
```

Comment:
Imports should be alphabetized. You can run `yarn run lint:app:ts`.

Reply:
This should also be up with 3rd party imports